apps/meteor/client/sidebar/footer/voip/VoipFooter.tsx

```diff
@@ -107,7 +107,7 @@ export const VoipFooter = ({
 								data-tooltip={holdTitle}
 								disabled={!isEnterprise}
 								icon={paused ? 'pause' : 'pause-unfilled'}
-								color={paused ? 'neutral-500' : 'info'}
+								color={paused ? 'disabled' : 'info'}
```

Comment:
here too.

Reply:
info is deprecated but still works good catch